src/dev-core/src/app/button/button.demo.ts

```diff
@@ -13,10 +14,22 @@ import '@clr/core/button';
   styleUrls: ['./button.demo.scss'],
 })
 export class ButtonDemoComponent {
+  form: FormGroup;
   disabled = true;
   myValidateBtnState = ClrLoadingState.DEFAULT;
   mySubmitBtnState = ClrLoadingState.DEFAULT;
 
+  constructor(private formBuilder: FormBuilder) {
+    this.form = this.formBuilder.group({
+      name: ['hello world'],
+    });
+  }
+
+  submit() {
+    console.log('Form Submit:', this.form.value);
```

Comment:
prob want to remove this console.log even though it's demo code? I think the alert below should be sufficient for testing / sanity check.

Reply:
removed